jstests/sharding/database_versioning_all_commands.js

```diff
@@ -866,65 +866,10 @@ const allTestCases = {
         },
         waitForFailPoint: {skip: "executes locally on mongos (not sent to any remote node)"},
         whatsmyuri: {skip: "executes locally on mongos (not sent to any remote node)"},
-    },
-<<<<<<< HEAD
-    updateRole: {skip: "always targets the config server"},
-    updateSearchIndex: {skip: "executes locally on mongos", conditional: true},
-    updateUser: {skip: "always targets the config server"},
-    updateZoneKeyRange: {skip: "not on a user database"},
-    usersInfo: {skip: "always targets the config server"},
-    validate: {
-        run: {
-            sendsDbVersion: true,
-            explicitlyCreateCollection: true,
-            command: function(dbName, collName) {
-                return {validate: collName};
-            },
-        }
-    },
-    validateDBMetadata: {
-        run: {
-            // validateDBMetadata is always broadcast to all shards.
-            sendsDbVersion: false,
-            explicitlyCreateCollection: true,
-            command: function(dbName, collName) {
-                return {validateDBMetadata: 1, apiParameters: {version: "1"}};
-            },
-        }
-    },
-    waitForFailPoint: {skip: "executes locally on mongos (not sent to any remote node)"},
-    whatsmyuri: {skip: "executes locally on mongos (not sent to any remote node)"},
 
-    // Percona commands
-    auditGetOptions: {skip: "executes locally on mongos (not sent to any remote node)"},
-||||||| 97867d5d209
-    updateRole: {skip: "always targets the config server"},
-    updateSearchIndex: {skip: "executes locally on mongos", conditional: true},
-    updateUser: {skip: "always targets the config server"},
-    updateZoneKeyRange: {skip: "not on a user database"},
-    usersInfo: {skip: "always targets the config server"},
-    validate: {
-        run: {
-            sendsDbVersion: true,
-            explicitlyCreateCollection: true,
-            command: function(dbName, collName) {
-                return {validate: collName};
-            },
-        }
-    },
-    validateDBMetadata: {
-        run: {
-            // validateDBMetadata is always broadcast to all shards.
-            sendsDbVersion: false,
-            explicitlyCreateCollection: true,
-            command: function(dbName, collName) {
-                return {validateDBMetadata: 1, apiParameters: {version: "1"}};
-            },
-        }
+        // Percona commands
+        auditGetOptions: {skip: "executes locally on mongos (not sent to any remote node)"},
```

Comment:
This seems wrong. The `auditGetOptions` should be in both `mongos` and `mongod`.